llama-index-core/pyproject.toml

```diff
@@ -46,7 +46,7 @@ name = "llama-index-core"
 packages = [{include = "llama_index"}]
 readme = "README.md"
 repository = "https://github.com/run-llama/llama_index"
-version = "0.12.1"
+version = "0.12.2-rc1"
```

Comment:
I don't think we want to make an official (pre) release yet, do we?

Reply:
Nope, not one bit 😅 this was an attempt at forcing a cache refresh on the CI

Reply:
ahh yeah that was pretty frustrating to deal with lol